eds/blocks/local-navigation/local-navigation.css

```diff
@@ -153,6 +159,15 @@
   padding-inline: 0;
 }
 
+.local-navigation calcite-button {
+  display: flex;
+  inline-size: 100%;
+  padding-inline: 30%;
+  align-items: center;
+  background-color: #202020;
+  padding-block: var(--space-4);
+}
```

Comment:
pretty sure we don't need any of this. When I comment out all this excpet padding it works fine.

Reply:
Was able to delete lines 164 and 166... otherwise mobile would break.